java-components/pom.xml

```diff
@@ -25,11 +25,12 @@
         <formatter-maven-plugin.version>2.18.0</formatter-maven-plugin.version>
         <impsort-maven-plugin.version>1.6.2</impsort-maven-plugin.version>
 
+        <assertj.version>3.23.1</assertj.version>
         <aws-url-connection-client.version>2.17.95</aws-url-connection-client.version>
         <aws-java-sdk-s3.version>1.12.169</aws-java-sdk-s3.version>
         <quarkus-jgit.version>1.2.0</quarkus-jgit.version>
         <quarkus-operator-sdk.version>3.0.7</quarkus-operator-sdk.version>
-        <gradle-tooling-api.version>7.4.2</gradle-tooling-api.version>
+        <gradle-tooling-api.version>7.3-20210825160000+0000</gradle-tooling-api.version>
```

Comment:
I don't understand what repository the 7.4.2 version came from. I don't see it on Maven central.

Reply:
https://github.com/stuartwdouglas/jvm-build-service/blob/940aef09401c286e8658691c2dca6bc379a48a48/java-components/adhoc-analyser-tool/pom.xml#L169

They stopped publishing to Maven central so you have to use their repo.

Reply:
OK, it seems that the repository just needs to be moved to `java-components/pom.xml`, then.

Reply:
Moved the declaration and updated to Gradle 7.5 in[12baa22](https://github.com/redhat-appstudio/jvm-build-service/pull/109/commits/12baa22afbea207b1a7727a3197746614c86b2fb).